.travis.yml

```diff
@@ -72,7 +72,7 @@ script:
   - git rev-parse HEAD
   - VERBOSE=1 make -k all
 
-notifications:
-  irc: "chat.freenode.net#radare"
-  on_success: never
-  on_failure: always
+#notifications:
```

Comment:
:_

Reply:
I will remove that particular commit later, right now I just commented that out to prevent the spamming on Telegram.

Reply:
pls bring this back to merge